.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    "env": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "indent": [
+            "error",
```

Comment:
This is throwing an error for me in Atom when using the ESLint plugin:

    /Users/agibson/Sites/viewsourceconf/.eslintrc.js: Configuration for rule "indent" is invalid: Severity should be one of the following: 0 = off, 1 = warning, 2 = error (you passed "error").

Should the values here (and below) be `2` instead of `error`?